src/platforms/esp32/components/avm_builtins/Kconfig

```diff
@@ -0,0 +1,59 @@
+#
+# This file is part of AtomVM.
+#
+# Copyright 2022 Davide Bettio <davide@uninstall.it>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+#
+
+menu "AtomVM Built-In Components"
```

Comment:
We can do this in the future, but I would like to make a top-level "AtomVM Configuration" menu, for other AtomVM settings.  For example, I plan to add settings for RTC, and I can imagine in the future we may want to add general VM-specific settings.

So just a comment.  Feel free to add a top-level menu, if you can.